src/DynamoManipulation/MousePointManipulator.cs

```diff
@@ -192,15 +187,13 @@ protected override List<(NodeModel inputNode, double amount)> InputNodesToUpdate
 
                 // When more than one input is connected to the same slider, this
                 // method will decompose the axis corresponding to each input.
-                using (var v = GetFirstAxisComponent(item.Value.Item1))
-                {
-                    var amount = offset.Dot(v);
+                var v = GetFirstAxisComponent(item.Value.Item1);
+                var amount = offset % v;
```

Comment:
this is the dot operator for primitive vector types?

Reply:
Yes.